### PR TITLE
Added metrics to cip-email

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

File: test/uk/gov/hmrc/cipemail/controllers/NotificationControllerSpec.scala [hunk 1]
```diff
@@ -36,7 +37,8 @@ class NotificationControllerSpec extends AnyWordSpec
 
   private val fakeRequest = FakeRequest()
   private val mockVerifyConnector: VerifyConnector = mock[VerifyConnector]
-  private val controller = new NotificationController(Helpers.stubControllerComponents(), mockVerifyConnector)
+  private val mockMetricsService: MetricsService = mock[MetricsService]
+  private val controller = new NotificationController(Helpers.stubControllerComponents(), mockVerifyConnector, mockMetricsService)
```
Comment (brdgeth) [inline test/uk/gov/hmrc/cipemail/controllers/NotificationControllerSpec.scala:42]: please add a mock verify to check the metrics service is called with correct string parameter.

File: test/uk/gov/hmrc/cipemail/controllers/VerifyControllerSpec.scala [hunk 2]
```diff
@@ -36,7 +37,8 @@ class VerifyControllerSpec extends AnyWordSpec
 
   private val fakeRequest = FakeRequest()
   private val mockVerifyConnector: VerifyConnector = mock[VerifyConnector]
-  private val controller = new VerifyController(Helpers.stubControllerComponents(), mockVerifyConnector)
+  private val mockMetricsService: MetricsService = mock[MetricsService]
+  private val controller = new VerifyController(Helpers.stubControllerComponents(), mockVerifyConnector, mockMetricsService)
```
Comment (brdgeth) [inline test/uk/gov/hmrc/cipemail/controllers/VerifyControllerSpec.scala:42]: please add a mock verify to check the metrics service is called with correct string parameter.

File: test/uk/gov/hmrc/cipemail/controllers/VerifyPasscodeControllerSpec.scala [hunk 3]
```diff
@@ -36,7 +37,8 @@ class VerifyPasscodeControllerSpec extends AnyWordSpec
 
   private val fakeRequest = FakeRequest()
   private val mockVerifyConnector: VerifyConnector = mock[VerifyConnector]
-  private val controller = new VerifyPasscodeController(Helpers.stubControllerComponents(), mockVerifyConnector)
+  private val mockMetricsService: MetricsService = mock[MetricsService]
+  private val controller = new VerifyPasscodeController(Helpers.stubControllerComponents(), mockVerifyConnector, mockMetricsService)
```
Comment (brdgeth) [inline test/uk/gov/hmrc/cipemail/controllers/VerifyPasscodeControllerSpec.scala:42]: please add a mock verify to check the metrics service is called with correct string parameter.

**Review by brdgeth — CHANGES_REQUESTED** [review]
add mockito verify lines x 3

File: test/uk/gov/hmrc/cipemail/controllers/NotificationControllerSpec.scala (hunk 1 above)
Comment (mar36tin) [reply test/uk/gov/hmrc/cipemail/controllers/NotificationControllerSpec.scala:42]: done

File: test/uk/gov/hmrc/cipemail/controllers/VerifyControllerSpec.scala (hunk 2 above)
Comment (mar36tin) [reply test/uk/gov/hmrc/cipemail/controllers/VerifyControllerSpec.scala:42]: done

File: test/uk/gov/hmrc/cipemail/controllers/VerifyPasscodeControllerSpec.scala (hunk 3 above)
Comment (mar36tin) [reply test/uk/gov/hmrc/cipemail/controllers/VerifyPasscodeControllerSpec.scala:42]: done

**Review by brdgeth — APPROVED** [review]

**Review by brdgeth — APPROVED** [review]

**Pull request merged**